src/server/user_service.py

```diff
@@ -0,0 +1,33 @@
+import cherrypy
+from database_management import COLLECTIONS
+from config import CHERRYPY_CONFIG_DEFAULT, WITHOUT_AUTHENTICATION
+from base64 import b64encode
+
+
+@cherrypy.expose
+class UserService:
+
+    @cherrypy.tools.json_out()
+    def GET(self, login, password):
+        with COLLECTIONS['users'] as col:
+            user_exists = list(col.find({'login': login, 'password': password}))
+            if user_exists:
+                auth_token = b64encode(bytes(f'{login}:{password}', 'utf-8')).decode('utf-8')
+                return {'auth_token': auth_token}
+            else:
+                raise cherrypy.HTTPError(404, 'User not found')
+
+    @cherrypy.tools.json_in()
+    def POST(self):
```

Comment:
Nie jestem pewien czy potrzebujemy POSTA dla użytkowników, bo z tego co wiem nie mamy żadnego super admina i w związku z tym każdy może dodawać nowych użytkowników. Może zostawmy to na razie do ręcznego wpisywania do bazy.

Reply:
Albo zostawmy tak jak jest, a później to zabezpieczymy autoryzacją poprzez kredencjały superadmina.

Reply:
Super admin na POST jest dobrym pomysłem - można to wprowadzić w kolejnej wersji.